src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c

```diff
@@ -317,12 +317,19 @@ int zocl_pcap_download_ioctl(struct drm_device *dev, void *data,
 	uint64_t primary_fw_off;
 	uint64_t primary_fw_len;
 	struct axlf_section_header *primaryHeader;
+	int ret = 0;
 
 	if (copy_from_user(&bin_obj, args->xclbin, sizeof(struct axlf)))
 		return -EFAULT;
 
 	if (memcmp(bin_obj.m_magic, "xclbin2", 8))
 		return -EINVAL;
+	
+	/* Check unique ID */
+	if (bin_obj.m_uniqueId == zdev->unique_id_last_bitstream) {
+		DRM_INFO("The XCLBIN already loaded. Don't need to reload.");
+		return ret;
```

Comment:
we don't really need a ret here, right? just return 0.

Reply:
Fixed. Removed this as it used only in one place.